app/scripts/modules/core/src/managed/graphql/graphql-sdk.ts

```diff
@@ -197,6 +205,20 @@ export interface MdMoniker {
   detail?: Maybe<Scalars['String']>;
 }
 
+export interface MdNotification {
```

Comment:
Just for my own learning, are these types auto-generated from the GraphQL schema?

Reply:
Yes